tests/integration/test_vulnerability_detector/test_scan_results/test_scan_nvd_feeds.py

```diff
@@ -23,14 +23,14 @@
 current_test_path = os.path.dirname(os.path.realpath(__file__))
 test_data_path = os.path.join(current_test_path, 'data')
 configurations_path = os.path.join(test_data_path, 'wazuh_nvd_configuration.yaml')
-vulnerabilities_data_path = os.path.join(test_data_path, 'nvd_vulnerabilities.json')
+vulnerabilities_data_path = os.path.join(test_data_path, 'nvd_vulnerabilities_1.json')
 
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
 SCAN_TIMEOUT = 40
 
 
 # Set configuration
-CUSTOM_NVD_FEED = 'custom_nvd_feed.json'
+CUSTOM_NVD_FEED = 'custom_nvd_feed_1.json'
```

Comment:
I saw that you are using this variable in several tests. It would be better if you define this variable in `vulnerability_detector` module and then import it in the tests.

Reply:
Done in 6e15914860aeff82cdeacbc887c9b91fe400ede1